### PR TITLE
prep version 0.2.1

### LINKED ISSUE #127: EDV (using BSync v0.2.0) related question
I was testing [EDV workflow](https://github.com/NREL/edv_synthetic_smart_meter_data) lately and found something that was missing in tag version `v0.2.0` but which was available in the current (as of today) `develop` branch. Summary below.

- EDV plan is to use BuildingSync-gem v0.2.0 (for now) for creating EP/OS models

- but when I run below on EDV's side,
`translator = BuildingSync::Translator.new(xml_file_path, out_path, epw_file_path, standard, false)`  

- I was getting the following error
`Error occurred while processing C:/Users/JKIM4/Documents/GitHub/edv-experiment-1/workflow_results/Add_Measured_Data_Files/Panther_office_Karla.xml with message: uninitialized constant BuildingSync::Generator
Did you mean?  Enumerator`

- was tracking down why it wasn't accepting BuildingSync::Generator and found out `translator.rb` wasn't including `require 'buildingsync/generator'` in v0.2.0 while it was there in `develop` branch as shown [here](https://github.com/BuildingSync/BuildingSync-gem/blob/fe698bbe97d9241e12e24437c27af4600cd864fd/lib/buildingsync/translator.rb#L41) 

- so was wondering,
  - if this difference was intended for v0.2.0
  - or if `require 'buildingsync/generator'` could also be added in v0.2.0

Let me know if I'm just at the wrong place for this issue or if you have any questions/comments/suggestions. Thanks!!

### LINKED ISSUE #130: Update purge measures method
Only remove measures from workflow that have SKIP argument explicitly set to True. Sometimes SKIP argument is missing: in that case we want to keep the measure in the workflow.

### PR DESCRIPTION
**Pull request by kflemin**
Fixes #127 
Fixes #130

### CODE REVIEW

**Review by nllong — APPROVED** [review]
looks great!

File: lib/buildingsync/makers/workflow_maker.rb [hunk 1]
```diff
@@ -518,15 +518,24 @@ def deep_copy_workflow
...
     # copy is made and the measures are configured for the specific scenario
+    # KAF: reworked to only delete measures with an explicit __SKIP__ == true 
+    # (sometimes measure don't have a skip at all, assume we want to keep those)
     def purge_skipped_from_workflow(workflow)
       non_skipped = []
       if !workflow.nil? && !workflow['steps'].nil? && workflow.key?('steps')
         workflow['steps'].each do |step|
-          if !step.nil? && step.key?('arguments') && !step['arguments'].nil?
-            if step['arguments'].key?('__SKIP__') && step['arguments']['__SKIP__'] == false
+          if !step.nil? && step.key?('arguments')
```
Comment (nllong) [inline lib/buildingsync/makers/workflow_maker.rb:527]: nice find and fix!

**Review by nllong — APPROVED** [review]
🤑

**Comment by JanghyunJK** [comment]
awesome!

**Pull request merged**